lib/element.js

```diff
@@ -161,6 +161,100 @@ ElementArrayFinder.prototype.all = function(locator) {
 };
 
 /**
+ * Get a slice of elements within the ElementArrayFinder.
+ * Negative indices are wrapped (i.e. -i means ith element from last)
+ * This does not actually retrieve the underlying element.
+ *
+ * Slice can be negative in which case we go backwards.
+ *
+ * Note that if step is positive and start > stop, an error is thrown
```

Comment:
I think it makes more sense to keep things in line with how `Array.prototype.slice` works in JavaScript or give this a different name.